tests/torch/test_compile_torch.py

```diff
@@ -1258,6 +1259,54 @@ def test_fancy_indexing_torch(model_object, default_configuration):
     compile_brevitas_qat_model(model, x, n_bits=4, configuration=default_configuration)
 
 
+@pytest.mark.parametrize(
+    "model_object",
+    [
+        pytest.param(MultiOutputModel),
+    ],
+)
+def test_multi_output(model_object, default_configuration):
+    """Test fancy indexing torch."""
+    model = model_object()
+    x = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+    y = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+    torch_result = model(x[[0]], y[[0]])
+
+    # Compare with simulation
+    quantized_module = compile_torch_model(
+        model, (x, y), n_bits=4, configuration=default_configuration
+    )
+    qm_result = quantized_module.forward(x[[0]], y[[0]])
+    simulation_result = quantized_module.forward(x[[0]], y[[0]], fhe="simulate")
+    assert isinstance(qm_result, tuple) and len(qm_result) == 2
+    assert isinstance(simulation_result, tuple) and len(simulation_result) == 2
+    for qm_res, sim_res in zip(qm_result, simulation_result):
+        assert isinstance(qm_res, numpy.ndarray)
+        assert isinstance(sim_res, numpy.ndarray)
+        assert (qm_res == sim_res).all()
+    assert isinstance(torch_result, tuple) and len(torch_result) == 2
+    for qm_res, trch_res in zip(qm_result, torch_result):
+        assert isinstance(qm_res, numpy.ndarray)
+        assert isinstance(trch_res, numpy.ndarray)
+        # very low tolerance because we use low-bit-width
+        assert numpy.isclose(qm_res, trch_res, atol=1e-1).all()
+
+    # Compare with high bit width quantized_module
+    quantized_module = build_quantized_module(
+        model,
+        (x, y),
+        n_bits=24,
+    )
+    qm_result = quantized_module.forward(x[[0]], y[[0]])
+    assert isinstance(qm_result, tuple) and len(qm_result) == 2
+    assert isinstance(torch_result, tuple) and len(torch_result) == 2
+    for qm_res, trch_res in zip(qm_result, torch_result):
+        assert isinstance(qm_res, numpy.ndarray)
+        assert isinstance(trch_res, numpy.ndarray)
+        # very low tolerance because we use low-bit-width
```

Comment:
better to have a comment starting with a capitalized letter ! also, better to have a space above a comment